react/Chip/index.jsx

```diff
@@ -3,30 +3,56 @@ import React from 'react'
 import styles from './styles.styl'
 
 class Chip extends React.PureComponent {
+  static defaultProps = {
+    component: 'div',
+    size: 'normal',
+    variant: 'normal',
+    theme: 'normal'
+  }
+
   render() {
-    const { children, className, ...restProps } = this.props
+    const {
+      children,
+      className,
+      rounded,
+      component: Component,
+      onClick,
+      disabled,
+      size,
+      variant,
+      theme,
+      ...restProps
+    } = this.props
+
     return (
-      <div
+      <Component
         className={cx(
           styles['c-chip'],
+          styles[`c-chip--${size}Size`],
+          styles[`c-chip--${variant}Variant`],
+          styles[`c-chip--${theme}Theme`],
           'u-breakword',
-          className,
-          this.constructor.className
+          {
+            [styles['c-chip--round']]: rounded,
+            'u-c-pointer': onClick && !disabled,
```

Comment:
I dont think we should use u-c-pointer here because it is an utility class that won't be overrided easily. Prefer not to use utility classes in library components.